src/main/java/com/codessquad/qna/web/users/UsersController.java

```diff
@@ -1,61 +1,59 @@
 package com.codessquad.qna.web.users;
 
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
-
-import java.util.ArrayList;
-import java.util.List;
+import org.springframework.web.bind.annotation.*;
 
 @Controller
+@RequestMapping("users")
 public class UsersController {
 
-    private final List<User> userList = new ArrayList<>();
+    @Autowired
+    private UserRepository userRepository;
 
-    @PostMapping("users")
-    public String createUser(String userId, String password, String name, String email) {
-        User createdUser = new User(userId, password, name, email);
-        userList.add(createdUser);
+    @PostMapping()
```

Comment:
전달할 매개변수가 없다면 괄호는 과감히 생략해 주세요.
```suggestion
    @PostMapping

Reply:
안붙여도 되는군요? 이 부분도 반영하겠습니다! 😄